test/ConfigResourceTest.php

```diff
@@ -402,4 +402,80 @@ public function testDeleteNonexistentKeyShouldDoNothing()
         $test = include $this->file;
         $this->assertEquals($expected, $test);
     }
+
+    public function testDataIsSorted()
+    {
+        // Arrange
+        $config = [
+            'z' => 1,
+            'a' => 2,
+        ];
+
+        $configResource = new ConfigResource($config, $this->file, new PhpArray());
+
+        // Act
+        $configResource->overWrite($config);
+
+        // Assert
+        $test = include $this->file;
+
+        self::assertEquals(2, current($test));
+        self::assertEquals(1, next($test));
+    }
+
+    public function testDataIsSortedRecursively()
```

Comment:
Rename this to `testDataIsSortedRecursivelyByKey()`.

Reply:
Done :+1: